src/routes/activity/repository.js

```diff
@@ -0,0 +1,25 @@
+const { db, errors } = require('../../shared')
+
+const { InsertActivityError } = errors.system
+
+const insertActivity = async (activity) => {
```

Comment:
This activity doesn't contains the auth data on this part. You can expect the auth object here
```suggestion
const insertActivity = async (activity, auth) => {
```